addons/pos_sale/models/pos_order.py

```diff
@@ -8,6 +8,14 @@ class PosOrder(models.Model):
     _inherit = 'pos.order'
 
     currency_rate = fields.Float("Currency Rate", compute='_compute_currency_rate', store=True, digits=(12, 6), readonly=True, help='The rate of the currency to the currency of rate applicable at the date of the order')
+    crm_team_id = fields.Many2one('crm.team', string="Sales Team")
+
+    @api.model
+    def create(self, values):
```

Comment:
we should  add a default function instead of overriding create, like for other values on the pos.order (pricelist, company,...) as:
```suggestion
    def _default_crm_team(self):
          return self._default_session().config_id.crm_team_id
```

Reply:
a default value function is a class method, so it is unaware of the object's session that is needed to set the team.

Reply:
so why are you calling '_default_session' to now which session to use to set the team?

Reply:
Actually both are needed, I guess you like the way I wrote it now better with set_default.

Reply:
@switch87 I'll just let the default function on the field 'crm_team_id' for now. And I'll create a task that will transform all the fake default values (depending on the session or config) like the _default_session, _default_pricelist, fiscal_position_id, ... in values added in the create.